src/widget/wlabel.cpp

```diff
@@ -148,3 +149,12 @@ void WLabel::fillDebugTooltip(QStringList* debug) {
     WBaseWidget::fillDebugTooltip(debug);
     *debug << QString("Text: \"%1\"").arg(text());
 }
+
+int WLabel::getHighlight() const {
+    return m_highlight;
+}
+
+void WLabel::setHighlight(int highlight) {
+    m_highlight = highlight;
+    emit highlightChanged(m_highlight);
```

Comment:
Early exit if the new value equals the current value? Not sure how often this happens.

Reply:
idk either. won't hurt though
(changed it for widgetgroup as well as part of the code is identical there)